spec/mongoid/document_spec.rb

```diff
@@ -541,7 +557,7 @@ class << self; attr_accessor :name; end
         end
 
         it 'returns all fields and _id' do
-          actual = church.as_json(compact: true)
+          actual = church.as_json(compact: true).compact
```

Comment:
Did you mean to remove the `compact: true` argument here?

Reply:
Oops yes. Fixed.